tests/common.go

```diff
@@ -2822,6 +2845,9 @@ func SetClusterContext(clusterConfigPath string) error {
 		if err != nil {
 			return fmt.Errorf("failed to switch to context. RefreshDriver (Node) Error: [%v]", err)
 		}
+	} else {
+		err = Inst().N.Init(node.InitOptions{
+			SpecDir: Inst().SpecDir})
```

Comment:
Why was this needed?

Reply:
This is needed in case we are running on cloud to spin up debug pods

Reply:
done ,added the comments